plugin/generator/generator.go

```diff
@@ -0,0 +1,314 @@
+package main
+
+import (
+	"bytes"
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+	"text/template"
+)
+
+var functions = []Function{
```

Comment:
@alowde this generator / upcoming change might be interesting for your work on complex config callbacks.